mindsdb/__main__.py

```diff
@@ -167,7 +167,7 @@ def close_api_gracefully(apis):
     # @TODO Backwards compatibility for tests, remove later
 
     if args.api is None:
-        api_arr = ['http', 'mysql']
+        api_arr = ['http', 'mysql', 'mongodb']
```

Comment:
I doubt that we need mondo server running by default

Reply:
Yes, let's just keep the http and mysql. Mongo should be optional

Reply:
Should be removed completely or can be controlled from config / environment variable ?

Reply:
Done removed